docs/source/index.rst

```diff
@@ -25,6 +25,7 @@ We appreciate any feedback and contributions to the project. If you want to cont
     Quickstart
     EquivalenceChecking
     CompilationFlowVerification
+    ParameterizedCircuits
```

Comment:
Is it possible that you missed uploading that file? I can't find it and it isn't shown in the docs.

Reply:
Indeed. Changed it back.